test/e2e/e2e_test.go

```diff
@@ -325,9 +325,17 @@ func verifyResourcesProvisionedForSignup(t *testing.T, awaitility *wait.Awaitili
 		UserAccountStatus: userAccount.Status,
 	}
 	_, err = hostAwait.WaitForMasterUserRecord(mur.Name,
-		wait.UntilMasterUserRecordHasConditions(provisioned()),
+		wait.UntilMasterUserRecordHasConditions(provisioned(), provisionedNotificationCRCreated()),
 		wait.UntilMasterUserRecordHasUserAccountStatuses(expectedEmbeddedUaStatus))
 	assert.NoError(t, err)
+
+	notification, err := hostAwait.WaitForNotification(userAccount.Name + "-provisioned")
+	assert.NoError(t, err)
+	require.NotNil(t, notification)
+	assert.Equal(t, notification.Name, userAccount.Name+"-provisioned")
+	assert.Equal(t, notification.Namespace, mur.Namespace)
+	assert.Equal(t, notification.Spec.Template, "userprovisioned")
+	assert.Equal(t, notification.Spec.UserID, userAccount.Spec.UserID)
```

Comment:
The first param is the expected value. The second is the actual one. Please swap them.

Reply:
fixed as part of: 4afd56d532ea1db81994c09a35511c8bc26e2b79